geode-core/src/distributedTest/java/org/apache/geode/management/JMXMBeanReconnectDUnitTest.java

```diff
@@ -116,7 +114,8 @@
   @Rule
   public DistributedRule distributedRule = new DistributedRule();
   @Rule
-  public SerializableTemporaryFolder temporaryFolder = new SerializableTemporaryFolder();
+  public SerializableTemporaryFolder temporaryFolder = new SerializableTemporaryFolder()
+      .copyTo(new File(getClass().getSimpleName())).when(FAILS);
```

Comment:
Remove the custom debugging and then file an additional PR that changes `SerializableTemporaryFolder` to have that behavior by default.

Reply:
Done